tests/plugins/test_store_metadata.py

```diff
@@ -15,26 +15,11 @@
 from copy import deepcopy
 from textwrap import dedent
 
-try:
-    import koji as koji
-except ImportError:
-    import inspect
-    import os
-    import sys
-
-    # Find our mocked koji module
-    import tests.koji as koji
-    mock_koji_path = os.path.dirname(inspect.getfile(koji.ClientSession))
-    if mock_koji_path not in sys.path:
-        sys.path.append(os.path.dirname(mock_koji_path))
-finally:
-    del koji
-    from atomic_reactor.plugins.exit_koji_promote import KojiPromotePlugin
```

Comment:
Aha... of course, we no longer need it in *this* module, because it doesn't need koji itself nor does the module under test.